setup.py

```diff
@@ -46,7 +46,7 @@
         'lxml',
         'beautifulsoup4',
         'requests',
-        'weasyprint'
+        'weasyprint<53'
```

Comment:
```suggestion
        'weasyprint'
```
I know why you use weasyprint<53, but you should not limit in setup.py, weasyprint depending of pango library. fix by installing weasyprint<53 after pip install .[dev]